src/components/universal/core/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import { Badge } from "components/universal";
+import { Flex } from "generated/panda/jsx";
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+type Story = StoryObj<typeof meta>;
+
+export const Solid: Story = {
+  render: () => <Badge variant="solid">Badge</Badge>,
+};
+export const Subtle: Story = {
```

Comment:
Update to `Default` and make this the first story when `defaultVariants` is removed, should not need to pass `variant` prop